### ISSUE TITLE
deployment manifest scheme missing asset fetchinng

### ISSUE THREAD

**Issue by mavenraven** [cloudfoundry/docs-bosh#76]
i.e. [this](https://github.com/cloudfoundry/bosh-notes/blob/master/asset-fetching.md) isn't documented on http://bosh.io/docs/deployment-manifest.html#releases

**Comment by dpb587-pivotal** [comment]
The [cloudfoundry/bosh-notes](https://github.com/cloudfoundry/bosh-notes) repo documents features which are currently being planned, discussed, and experimented with so they're generally not features which bosh supports yet - asset fetching is one of those features. Asset fetching does currently work, and is supported by, the [`bosh-init`](http://bosh.io/docs/using-bosh-init.html) command, just not in a regular bosh manifest which is what that page is describing.

The bosh.io docs are limited to documenting official, released features.

**Comment by cppforlife** [comment]
@mavenraven yup it's been implemented, just not documented yet. on my todo list.

**Comment by mavenraven** [comment]
@dpb587-pivotal It actually does work for a regular bosh manifest in the latest version of bosh.

@cppforlife Where is the bosh.io repo? I'll send you a PR if I have some time.

**Comment by bosh-ci-push-pull** [comment]
docs-bosh repo is used for docs: https://github.com/cloudfoundry/docs-bosh
